README.md

```diff
@@ -117,6 +117,33 @@ Get your AccessKeyID and SecretAccessKey by following [AWS Credentials Guide](ht
 mc config host add s3 https://s3.amazonaws.com BKIKJAA5BMMU2RHO6IBB V7f1CwQqAcwo80UEIJEjc5gVQUSSx5ohQ9GSrr12
 ```
 
+NOTE: As an IAM user on Amazon S3 you need to make sure user has full access or make sure following policy is set for your IAM user
```

Comment:
"...you need to make sure the user has full access to the bucket or make sure the following..."

Reply:
Next time use https://help.github.com/articles/incorporating-feedback-in-your-pull-request/#applying-a-suggested-change makes it easy to merge the suggested changes.